quickcomm/models.py

```diff
@@ -256,6 +255,34 @@ class InboxType(models.TextChoices):
     def __str__(self):
         return f"{self.author.__str__()}'s inbox contains {self.content_object.__str__()}"
 
+class CommentLike(models.Model):
+    """A comment like is a relationship between an author and a comment."""
+
+    author = models.ForeignKey(Author, on_delete=models.CASCADE)
+    comment = models.ForeignKey(Comment, on_delete=models.CASCADE)
+
+    def save(self, *args, **kwargs):
+        saved = super(CommentLike, self).save(*args, **kwargs)
+        # When we save a comment like, we also need to create an inbox post for the
+        # author of the post.
+
+        # skip inbox logic for remote authors
+        #if self.author.is_remote:
+         #   return saved
+
+        Inbox.objects.create(content_object=self, author=self.comment.post.author, inbox_type=Inbox.InboxType.COMMENTLIKE)
```

Comment:
This line means that we're sending the inbox item to the author of the post, not the author of the comment. So if Mary comments on John's post, then John likes that comment, he will get the comment like in his inbox, not Mary.

Should probably change to just `author=self.comment.author`